commands/cmdinfo.js

```diff
@@ -1,21 +1,26 @@
 const fs = require('fs');
 module.exports = {
     run: async (client, msg, args) => {
-        if (!args[0]) return msg.channel.send(`\\❌ Invalid usage. Expected usage:\n\n\`\`\`${client.config.prefix + module.exports.meta.aliases[0] + ' ' + module.exports.meta.usage}\`\`\``);
+        if (args.length !== 1) return msg.channel.send(`\\❌ Invalid usage. Expected usage:\n\n\`\`\`${client.config.prefix + module.exports.meta.aliases[0] + ' ' + module.exports.meta.usage}\`\`\``);

```

Comment:
args[0] could be undefined, use length checking instead.

Reply:
This isn't Java lol

Reply:
what makes you think it is?

Reply:
In JS **you** can use `!args[0]` and it will check if that index is existent or not - in Java, that will cause a Null Pointer Exception.

Reply:
Just saying it doesn't fit in with the rest of the code :man_shrugging:

Reply:
What about if the user provides more than 1 argument? :O

Reply:
Nvm it has argument length checking somewhere